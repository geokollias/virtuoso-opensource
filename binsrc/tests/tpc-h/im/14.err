Starting Impala Shell without Kerberos authentication
Connected to ip-10-0-1-251.ec2.internal:21000
Server version: impalad version 2.1.3-cdh5 RELEASE (build 20816e26a150d20d2c92c470aa40b342521b243e)
Query: use TPCH
Query: select 100.00 * sum(case
		when p_type like 'PROMO%'
			then l_extendedprice * (1 - l_discount)
		else 0
	end) / sum(l_extendedprice * (1 - l_discount)) as promo_revenue
from
	lineitem,
	part 
where
	l_partkey = p_partkey
	and l_shipdate >= '1995-09-01'
	and l_shipdate < cast ('1995-09-01' as timestamp) + INTERVAL 1 MONTH
Fetched 1 row(s) in 25.40s
