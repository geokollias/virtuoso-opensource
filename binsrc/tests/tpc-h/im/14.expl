[?1034h+--------------------------------------------------------------------------------------------------------------------------------------------------------+
| Explain String                                                                                                                                         |
+--------------------------------------------------------------------------------------------------------------------------------------------------------+
| Estimated Per-Host Requirements: Memory=578.61MB VCores=2                                                                                              |
| WARNING: The following tables are missing relevant table and/or column statistics.                                                                     |
| tpch.part                                                                                                                                              |
|                                                                                                                                                        |
| 06:AGGREGATE [FINALIZE]                                                                                                                                |
| |  output: sum:merge(CASE WHEN p_type LIKE 'PROMO%' THEN l_extendedprice * (1 - l_discount) ELSE 0 END), sum:merge(l_extendedprice * (1 - l_discount)) |
| |  hosts=1 per-host-mem=unavailable                                                                                                                    |
| |  tuple-ids=2 row-size=16B cardinality=1                                                                                                              |
| |                                                                                                                                                      |
| 05:EXCHANGE [UNPARTITIONED]                                                                                                                            |
| |  hosts=1 per-host-mem=unavailable                                                                                                                    |
| |  tuple-ids=2 row-size=16B cardinality=1                                                                                                              |
| |                                                                                                                                                      |
| 03:AGGREGATE                                                                                                                                           |
| |  output: sum(CASE WHEN p_type LIKE 'PROMO%' THEN l_extendedprice * (1 - l_discount) ELSE 0 END), sum(l_extendedprice * (1 - l_discount))             |
| |  hosts=1 per-host-mem=10.00MB                                                                                                                        |
| |  tuple-ids=2 row-size=16B cardinality=1                                                                                                              |
| |                                                                                                                                                      |
| 02:HASH JOIN [INNER JOIN, BROADCAST]                                                                                                                   |
| |  hash predicates: p_partkey = l_partkey                                                                                                              |
| |  hosts=1 per-host-mem=226.61MB                                                                                                                       |
| |  tuple-ids=1,0 row-size=56B cardinality=20000000                                                                                                     |
| |                                                                                                                                                      |
| |--04:EXCHANGE [BROADCAST]                                                                                                                             |
| |  |  hosts=1 per-host-mem=0B                                                                                                                          |
| |  |  tuple-ids=0 row-size=36B cardinality=6000379                                                                                                     |
| |  |                                                                                                                                                   |
| |  00:SCAN HDFS [tpch.lineitem, RANDOM]                                                                                                                |
| |     partitions=1/1 files=82 size=20.20GB                                                                                                             |
| |     predicates: l_shipdate >= '1995-09-01', l_shipdate < CAST('1995-09-01' AS TIMESTAMP) + INTERVAL 1 MONTH                                          |
| |     table stats: 600037902 rows total                                                                                                                |
| |     column stats: all                                                                                                                                |
| |     hosts=1 per-host-mem=352.00MB                                                                                                                    |
| |     tuple-ids=0 row-size=36B cardinality=6000379                                                                                                     |
| |                                                                                                                                                      |
| 01:SCAN HDFS [tpch.part, RANDOM]                                                                                                                       |
|    partitions=1/1 files=3 size=648.75MB                                                                                                                |
|    table stats: 20000000 rows total                                                                                                                    |
|    columns missing stats: p_type                                                                                                                       |
|    hosts=1 per-host-mem=176.00MB                                                                                                                       |
|    tuple-ids=1 row-size=20B cardinality=20000000                                                                                                       |
+--------------------------------------------------------------------------------------------------------------------------------------------------------+
