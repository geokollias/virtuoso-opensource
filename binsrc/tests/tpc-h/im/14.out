[?1034h+-------------------+
| promo_revenue     |
+-------------------+
| 16.64035743325395 |
+-------------------+
